<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prymatex Text Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/icons/Prymatex_Logo.png</normaloff>:/icons/icons/Prymatex_Logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="PMXSplitTabWidget" name="splitTabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/actions/document-open-recent.png</normaloff>:/icons/actions/document-open-recent.png</iconset>
     </property>
     <addaction name="actionOpenAllRecentFiles"/>
     <addaction name="actionRemoveAllRecentFiles"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionNewFromTemplate"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSaveAll"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="actionCloseOthers"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuGutter">
     <property name="title">
      <string>Gutter</string>
     </property>
     <addaction name="actionShowFolding"/>
     <addaction name="actionShowBookmarks"/>
     <addaction name="actionShowLineNumbers"/>
    </widget>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>Font</string>
     </property>
     <addaction name="actionZoomIn"/>
     <addaction name="actionZoomOut"/>
    </widget>
    <addaction name="menuFont"/>
    <addaction name="menuGutter"/>
    <addaction name="separator"/>
    <addaction name="actionShowWhitespace"/>
    <addaction name="actionShowIndentation"/>
    <addaction name="actionShowEndOfLine"/>
   </widget>
   <widget class="QMenu" name="menuText">
    <property name="title">
     <string>&amp;Text</string>
    </property>
    <widget class="QMenu" name="menuConvert">
     <property name="title">
      <string>Convert</string>
     </property>
     <addaction name="actionToUppercase"/>
     <addaction name="actionToLowercase"/>
     <addaction name="actionToTitlecase"/>
     <addaction name="actionToOppositeCase"/>
     <addaction name="separator"/>
     <addaction name="actionSpacesToTabs"/>
     <addaction name="actionTabToSpaces"/>
     <addaction name="separator"/>
     <addaction name="actionTranspose"/>
    </widget>
    <addaction name="menuConvert"/>
    <addaction name="separator"/>
    <addaction name="actionExecute"/>
    <addaction name="actionFilterThroughCommand"/>
   </widget>
   <widget class="QMenu" name="menuPanels">
    <property name="title">
     <string>&amp;Panels</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>&amp;Navigation</string>
    </property>
    <addaction name="actionToggleBookmark"/>
    <addaction name="actionNextBookmark"/>
    <addaction name="actionPreviousBookmark"/>
    <addaction name="actionRemoveAllBookmarks"/>
    <addaction name="separator"/>
    <addaction name="actionNextTab"/>
    <addaction name="actionPreviousTab"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionReport_Bug"/>
    <addaction name="actionTranslate_this_application"/>
    <addaction name="actionProjectHomepage"/>
    <addaction name="actionRead_documentation"/>
    <addaction name="separator"/>
    <addaction name="actionTakeScreenshot"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout_this_application"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuSelect">
     <property name="title">
      <string>Select</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/actions/edit-select-all.png</normaloff>:/icons/actions/edit-select-all.png</iconset>
     </property>
     <addaction name="actionSelectWord"/>
     <addaction name="actionSelectLine"/>
     <addaction name="actionSelectParagraph"/>
     <addaction name="actionSelectEnclosingBrackets"/>
     <addaction name="actionSelectCurrentScope"/>
     <addaction name="actionSelectAll"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="menuSelect"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFindReplace"/>
   </widget>
   <widget class="QMenu" name="menuBundles">
    <property name="title">
     <string>&amp;Bundles</string>
    </property>
    <addaction name="actionShowBundleEditor"/>
    <addaction name="actionSelectBundleItem"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="actionShowMenus"/>
    <addaction name="actionShowStatus"/>
    <addaction name="actionFullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuPanels"/>
   <addaction name="menuText"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuBundles"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/document-new.png</normaloff>:/icons/actions/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/document-open.png</normaloff>:/icons/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/document-save.png</normaloff>:/icons/actions/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/document-save-as.png</normaloff>:/icons/actions/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/document-save-all.png</normaloff>:/icons/actions/document-save-all.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/dialog-close.png</normaloff>:/icons/actions/dialog-close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionCloseOthers">
   <property name="text">
    <string>Close Others</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/application-exit.png</normaloff>:/icons/actions/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/edit-undo.png</normaloff>:/icons/actions/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/edit-redo.png</normaloff>:/icons/actions/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/edit-copy.png</normaloff>:/icons/actions/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/edit-cut.png</normaloff>:/icons/actions/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/edit-paste.png</normaloff>:/icons/actions/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/edit-find.png</normaloff>:/icons/actions/edit-find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/configure.png</normaloff>:/icons/actions/configure.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionShowMenus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Menus</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>:/actions/resources/actions/zoom-in.png</normaloff>:/actions/resources/actions/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>:/actions/resources/actions/zoom-out.png</normaloff>:/actions/resources/actions/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionShowLineNumbers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Line Numbers</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionShowFolding">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Foldings</string>
   </property>
   <property name="shortcut">
    <string>Shift+F10</string>
   </property>
  </action>
  <action name="actionShowBookmarks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bookmarks</string>
   </property>
   <property name="shortcut">
    <string>Alt+F10</string>
   </property>
  </action>
  <action name="actionToUppercase">
   <property name="text">
    <string>To Uppercase</string>
   </property>
  </action>
  <action name="actionToLowercase">
   <property name="text">
    <string>To Lowercase</string>
   </property>
  </action>
  <action name="actionToTitlecase">
   <property name="text">
    <string>To Titlecase</string>
   </property>
  </action>
  <action name="actionToOppositeCase">
   <property name="text">
    <string>To Opposite Case</string>
   </property>
  </action>
  <action name="actionTranspose">
   <property name="text">
    <string>Transpose</string>
   </property>
  </action>
  <action name="actionTabToSpaces">
   <property name="text">
    <string>Tab to Spaces</string>
   </property>
  </action>
  <action name="actionSpacesToTabs">
   <property name="text">
    <string>Spaces to Tabs</string>
   </property>
  </action>
  <action name="actionFilterThroughCommand">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/actions/view-filter.png</normaloff>:/resources/actions/view-filter.png</iconset>
   </property>
   <property name="text">
    <string>Filter Through Command</string>
   </property>
   <property name="iconText">
    <string>Filter Through Command</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="text">
    <string>Execute Line/Selection</string>
   </property>
  </action>
  <action name="actionToggleBookmark">
   <property name="text">
    <string>Toggle Bookmark</string>
   </property>
   <property name="shortcut">
    <string>Meta+F12</string>
   </property>
  </action>
  <action name="actionNextBookmark">
   <property name="text">
    <string>Next Bookmark</string>
   </property>
   <property name="shortcut">
    <string>Meta+Alt+F12</string>
   </property>
  </action>
  <action name="actionPreviousBookmark">
   <property name="text">
    <string>Previous Bookmark</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+F12</string>
   </property>
  </action>
  <action name="actionRemoveAllBookmarks">
   <property name="text">
    <string>Remove All Bookmarks</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+F12</string>
   </property>
  </action>
  <action name="actionNextTab">
   <property name="text">
    <string>Next Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgDown</string>
   </property>
  </action>
  <action name="actionPreviousTab">
   <property name="text">
    <string>Previous Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgUp</string>
   </property>
  </action>
  <action name="actionReport_Bug">
   <property name="text">
    <string>Report &amp;Bug</string>
   </property>
  </action>
  <action name="actionTranslate_this_application">
   <property name="text">
    <string>&amp;Translate this application</string>
   </property>
  </action>
  <action name="actionProjectHomepage">
   <property name="text">
    <string>Project &amp;Homepage</string>
   </property>
  </action>
  <action name="actionTakeScreenshot">
   <property name="text">
    <string>Take &amp;Screenshot</string>
   </property>
  </action>
  <action name="actionAbout_this_application">
   <property name="text">
    <string>&amp;About this application</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionShowBundleEditor">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Bundle Editor</string>
   </property>
  </action>
  <action name="actionSelectBundleItem">
   <property name="text">
    <string>Select Bundle Item</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+T</string>
   </property>
  </action>
  <action name="actionNewFromTemplate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/actions/document-new.png</normaloff>:/icons/actions/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New From Template</string>
   </property>
   <property name="shortcut">
    <string>Meta+Shift+N</string>
   </property>
  </action>
  <action name="actionShowWhitespace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Whitespace</string>
   </property>
  </action>
  <action name="actionShowIndentation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Indentation</string>
   </property>
  </action>
  <action name="actionShowEndOfLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show End of Line</string>
   </property>
  </action>
  <action name="actionRead_documentation">
   <property name="text">
    <string>Read &amp;documentation</string>
   </property>
  </action>
  <action name="actionSelectWord">
   <property name="text">
    <string>&amp;Word</string>
   </property>
  </action>
  <action name="actionSelectLine">
   <property name="text">
    <string>&amp;Line</string>
   </property>
  </action>
  <action name="actionSelectParagraph">
   <property name="text">
    <string>&amp;Paragraph</string>
   </property>
  </action>
  <action name="actionSelectEnclosingBrackets">
   <property name="text">
    <string>Enclosing &amp;Brackets</string>
   </property>
  </action>
  <action name="actionSelectCurrentScope">
   <property name="text">
    <string>Current &amp;Scope</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>&amp;All</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionShowStatus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Status</string>
   </property>
  </action>
  <action name="actionFindReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionOpenAllRecentFiles">
   <property name="text">
    <string>Open All Recent Files</string>
   </property>
  </action>
  <action name="actionRemoveAllRecentFiles">
   <property name="text">
    <string>Remove All Recent Files</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PMXSplitTabWidget</class>
   <extends>QTabWidget</extends>
   <header>prymatex.gui.central.splitter</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
